website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
I find `require.js` included in a number of node modules. It's not obvious to me that one or another of them is the source of our `require` global.

```
$ find . -name require.\*js
./node_modules/babel-core/node_modules/babel-register/node_modules/source-map-support/amd-test/require.js
./node_modules/babel-core/node_modules/json5/lib/require.js
./node_modules/babel-core/node_modules/json5/test/require.js
./node_modules/babel-loader/node_modules/babel-core/node_modules/source-map-support/amd-test/require.js
./node_modules/babel-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/babel-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/c3/htdocs/js/require.js
./node_modules/c3/node_modules/d3/node_modules/jsdom/node_modules/request/node_modules/har-validator/node_modules/is-my-json-valid/require.js
./node_modules/css-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/css-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/exports-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/exports-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/file-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/file-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/font-awesome-webpack/node_modules/css-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/font-awesome-webpack/node_modules/css-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/font-awesome-webpack/node_modules/less-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/font-awesome-webpack/node_modules/less-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/font-awesome-webpack/node_modules/style-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/font-awesome-webpack/node_modules/style-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/historyjs/node_modules/buildr/node_modules/less/build/require.js
./node_modules/html-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/html-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/imports-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/imports-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/karma/node_modules/chokidar/node_modules/fsevents/node_modules/is-my-json-valid/require.js
./node_modules/karma-sauce-launcher/node_modules/wd/node_modules/request/node_modules/har-validator/node_modules/is-my-json-valid/require.js
./node_modules/karma-webpack/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/karma-webpack/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/less/node_modules/request/node_modules/har-validator/node_modules/is-my-json-valid/require.js
./node_modules/style-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/style-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/url-loader/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/url-loader/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/webpack/node_modules/loader-utils/node_modules/json5/lib/require.js
./node_modules/webpack/node_modules/loader-utils/node_modules/json5/test/require.js
./node_modules/webpack/node_modules/watchpack/node_modules/chokidar/node_modules/fsevents/node_modules/is-my-json-valid/require.js
./website/static/vendor/bower_components/raven-js/plugins/require.js
```